adr/util/config.py

```diff
@@ -0,0 +1,40 @@
+import yaml
+
+
+class Configuration(object):
+    __slots__ = [
+        "url",
+        "verbose",
+        "debug",
+        "debug_url",
+        "format"
+    ]
+
+    def __init__(self, yml_file=None):
+        # No file: Empty config
+        if yml_file is None:
+            self.update(None, False, False, "json")
+            return
+        with open(yml_file, 'r') as content:
+            cfg = yaml.load(content)
+            self.update(cfg["url"], cfg["verbose"], cfg["debug"], cfg["format"])
```

Comment:
This is closer, but it would be better to not hard code the parameter names. E.g:

    for key, value in cfg.items():
        setattr(self, key, value)

The `__slots__` will make sure only allowed keys exist in the config file. It might also be good to handle default values in here (in case config.yml has some parameters but not others.